### PR TITLE
BREAKING CHANGE: Uncontrolled Tabs

### PR DESCRIPTION
**Pull request by literat**
Changing default behavior from uncontrolled to controlled components.
Setting this as the main concept of our components where we should provide controlled components as default and additionally provide an UncontrolledComponent for better DX.

### CODE REVIEW

**Comment by literat** [comment]
- [ ] Add some links about controlled vs uncontrolled components
- [ ] Revise Tabs readme - is it understandable? 
- [ ] Add some tests

File: packages/web-react/src/components/Tabs/stories/UncontrolledTabs.tsx [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+      <TabItem forTab={2}>Item</TabItem>
+    </TabList>
+    <TabContent>
+      <TabPane tabId={1}>Pane 1</TabPane>
+      <TabPane tabId={2}>Pane 2</TabPane>
+    </TabContent>
+  </UncontrolledTabs>
+);
+
+export default Story;
```
Comment (tomassychra) [inline packages/web-react/src/components/Tabs/stories/UncontrolledTabs.tsx:23]: Import this story to stories, thx

Comment (literat) [reply packages/web-react/src/components/Tabs/stories/UncontrolledTabs.tsx:23]: Thanks, I did not notice. Fixed.

**Review by tomassychra — APPROVED** [review]

**Review by vmilersky — APPROVED** [review]

File: packages/web-react/src/components/Tabs/README.md [hunk 2]
```diff
@@ -6,7 +6,13 @@ groups of information in tabbable regions.
 ## Tab
 
 ```jsx
-<Tabs defaultSelectedTab={1}>
+const [selectedTabId, setState] = useState(1);
```
[the remark this answers is not in the document]
Comment (literat) [reply packages/web-react/src/components/Tabs/README.md:9]: Is this ok for you? `const { selectedTabId, selectTab } = useTab(defaultSelectedTab);`

And also thanks for this catch up. I really missed it. I agree with you and rename it.

**Pull request merged**